lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -1173,6 +1177,37 @@ instance NFData StartTime
 newtype ProtocolMagic = ProtocolMagic Int32
     deriving (Generic, Show)
 
+{-------------------------------------------------------------------------------
+                                   Network
+-------------------------------------------------------------------------------}
+
+-- | Network-Time Protocol status.
+--
+-- TODO: Use a NTP-client to query some known NTP server and figure out by how
+-- much is machine's clock drifting.
+-- An existing NTP client implementation exists on `cardano-sl` and is fairly
+-- decoupled from the rest of the code:
+--
+-- https://github.com/input-output-hk/cardano-sl/blob/1a792d7cd0f0c93a0f0c28f66372bce3c3808dbd/networking/src/Ntp/Client.hs
+--
+-- The idea would be to extract that piece of code into a separate package that
+-- we could re-use to implement this functionality.
+data NtpStatus
+    = NtpUnavailable
+    | NtpPending
```

Comment:
What do the states `NtpUnavailable` and `NtpPending` mean?

Perhaps we can write Haddock documentation comments for each of these?

Reply:
Good point.